tests/test_templates.py

```diff
@@ -16,12 +14,16 @@
 from test_project import redirect_stdout, redirect_stderr
 
 
-class BaseTemplateTest(object):
+class TestBaseTemplate(object):
     project_class = signac.Project
-    env = None
+    envm = None
```

Comment:
Is there a good reason for this name change? If not, please revert it.

Reply:
Actually, no, I will correct it right away.